doc/metric-dev.rst

```diff
@@ -0,0 +1,227 @@
+
+For example, the PF1 metric is written as:
+
+.. code-block:: yaml
+
+    PF1:
+      # ...
+      specs:
+        - level: design
+          value: 10.0
+          units: ''
+          dependencies:
+            - PA2
+
+This dependency means that a measurement being compared against the 'design' specification of PF1 must use the PA2 'design' specification level as a parameter.
+
+Note that we only need to name the metric itself, the measurement framework will automatically find the equivalent specification in the dependent metric based on matching the level and filter.
+
+Creating Metric Objects in Python
+=================================
+
+Within Python, metrics are represented by instances of the :class:`lsst.validate.drp.base.Metric` class.
+
+A metric object is built from a YAML definition with the :meth:`lsst.validate.drp.base.Metric.fromYaml` class method.
+:meth:`~lsst.validate.drp.base.Metric.fromYaml` takes the metric name and either the path of a metric YAML file (``yamlPath`` keyword argument) or a pre-parsed YAML object (``yamlDoc`` keyword argument).
+
+For example:
+
+.. code-block:: python
+
+   import os
+   from lsst.utils import getPackageDir
+   from lsst.validate.drp.base import Metric
+   yamlPath = os.path.join(getPackageDir('validate_drp'),
+                           'metrics.yaml')
+   am1 = Metric.fromYaml('AM1', yamlPath=yamlPath)
+
+Checking a Measurement Against a Specification
+==============================================
+
+Ultimately, a metric object is most valuable in validating a measurement against a specification.
+For this, use the :meth:`lsst.validate.drp.base.Metric.checkSpec` method:
+
+
+.. code-block:: python
+
+   measuredValue = 2.  # hypothetical measured value
+   am1.checkSpec(measuredValue, 'design')
+
+The last statement will return ``True`` if the measured value fulfills the 'design' specification.
+If a specification is bandpass dependent, the bandpass needs to be passed to the ``bandpass`` keyword argument of :meth:`~lsst.validate.drp.base.Metric.checkSpec`.
+
+In :doc:`measurement-dev` we describe how to make measurements with the ``validate_drp`` API.
+
+Accessing Specification Objects of a Metric
+===========================================
+
+Since some measurements need to know about the specification levels of a metric, metrics provide a :meth:`~lsst.validate.drp.base.Metric.getSpec` method to resolve and retrieve a specification level.
+For example:
+
+.. code-block:: python
+
+   designSpec = am1.getSpec('design')
+
+If specification levels are bandpass-dependent, the bandpass can be provided with the ``bandpass`` keyword argument.
+
+The properties of a specification are retrieved through attributes:
+
+.. code-block:: python
+
+   designSpec.value
+   designSpec.units
+   designSpec.label
+   designSpec.bandpasses
+   designSpec.latex_units  # units marked up as LaTeX math
```

Comment:
Not sure I 💯 follow here. Here's the implementation: 

```
    @property
    def latex_units(self):
        """Units as a LateX string, wrapped in ``$``."""
        if self.units != '':
            fmtr = astropy.units.format.Latex()
            return fmtr.to_string(self.astropy_units)
        else:
            return ''
```

(https://github.com/lsst/validate_drp/blob/u/sqre/measurement-api/python/lsst/validate/drp/base.py#L201-L207)

In other words, `latex_units` is a read-only property that uses `astropy.units`.

So is this what you mean, just as a property rather than a method?